From 67c8c3ebd67c9137e78521d844506c52bcde56b1 Mon Sep 17 00:00:00 2001
From: Deepak Kumar <quic_deepaku@quicinc.com>
Date: Wed, 19 Jun 2024 12:39:12 +0530
Subject: [PATCH] OpenWRT: refresh 205-backtrace_module_info.patch

Fix compilation error due to some of the changes in structure module.

Change-Id: Idb1f4075bd0e05d0222cb9ccfc572e0f509607af
Signed-off-by: Deepak Kumar <quic_deepaku@quicinc.com>
---
 lib/vsprintf.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/vsprintf.c b/lib/vsprintf.c
index 524af575868b..542b6da79a0a 100644
--- a/lib/vsprintf.c
+++ b/lib/vsprintf.c
@@ -1010,8 +1010,8 @@ char *symbol_string(char *buf, char *end, void *ptr,
 	mod = __module_address(value);
 	if (mod)
 		snprintf(sym + len, sizeof(sym) - len, " [%s@%p+0x%x]",
-			 mod->name, mod->core_layout.base,
-			 mod->core_layout.size);
+			mod->name, mod->mem[MOD_TEXT].base,
+			mod->mem[MOD_TEXT].size);
 #endif
 	return string(buf, end, sym, spec);
 }
-- 
2.34.1

